third_party/python/Modules/_testcapimodule.c

```diff
@@ -4112,7 +4112,7 @@ pyobject_malloc_without_gil(PyObject *self, PyObject *args)
 
     Py_RETURN_NONE;
 }
-
+#ifdef USE_TRACEMALLOC
```

Comment:
Not worth ifdef'ing this out. We use `-ffunction-sections` so it'll be pruned by the linker.